continuous_integration/Dockerfile

```diff
@@ -1,17 +1,7 @@
-FROM sequenceiq/hadoop-docker:2.7.1
+FROM mdurant/docker_images:hadoop
```

Comment:
They are different tags, which is not necessarily the same as version (e.g., ubuntu:trusty gets you the latest, security updated version of trusty, not some specific release). I suppose dockerhub allows multiple entries to pull from the same github repo, would it be better to call it mdurant/dev and mdurant/hadoop (:latest assumed)?